ruby/ql/src/queries/security/cwe-116/IncompleteSanitization.ql

```diff
@@ -136,7 +136,7 @@ DataFlow::CallNode getAMethodCall(StringSubstitutionCall call) {
       // post-update receiver to the receiver of another method call?
       call.isDestructive() and
       exists(DataFlow::PostUpdateNode post | post.getPreUpdateNode() = call.getReceiver() |
-        post.(DataFlow::LocalSourceNode).flowsTo(receiver)
+        DataFlow::localFlow(post, receiver)
```

Comment:
May as well just be `DataFlow::localFlow(call.getReceiver(), receiver)`.

Reply:
I'm not convinced that's right – we want flow from the post-update node, not the pre-update node. The test fails with that change.

Reply:
Local flow should really not be concerned with post-update nodes, as that is a feature of the global data flow library used only in connection with field-flow. I am surprised that the test does not pass.